packages/relay/src/services/utils/IpfsHelper.ts

```diff
@@ -1,4 +1,5 @@
 import type { Helia } from '@helia/interface'
+import { ethers } from 'hardhat'
```

Comment:
You can remove this line if not use `ethers`

Reply:
fixed